configure

```diff
@@ -750,13 +751,11 @@ def cc_macros(cc=None):
                          stdout=subprocess.PIPE,
                          stderr=subprocess.PIPE)
   except OSError:
-    print('''Node.js configure error: No acceptable C compiler found!
+    error('''No acceptable C compiler found!
 
-        Please make sure you have a C compiler installed on your system and/or
-        consider adjusting the CC environment variable if you installed
-        it in a non-standard prefix.
-        ''')
-    sys.exit()
+       Please make sure you have a C compiler installed on your system and/or
+       consider adjusting the CC environment variable if you installed
+       it in a non-standard prefix.''')
```

Comment:
same here re: sys.exit()

Reply:
@MylesBorins This is in `def error(msg)` and ` sys.exit(1)` is included in it.

Reply:
thanks for explaining 😄